controllers/discovery/searchTopic.js

```diff
@@ -1,6 +1,6 @@
 const {QueryTypes} = require('sequelize');
-
-const {sequelize} = require('../../databases/models');
+const {User, sequelize} = require('../../databases/models');
+const UsersFunction = require('../../databases/functions/users');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `User` model and `UsersFunction` is added correctly. However, it's important to ensure that the `UsersFunction.findAnonymousUserId` method exists and works as expected.
```diff
- const {sequelize} = require('../../databases/models');
+ const {User, sequelize} = require('../../databases/models');
+ const UsersFunction = require('../../databases/functions/users');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->